config/main.py

```diff
@@ -3916,6 +3916,166 @@ def del_user(db, user):
             click.echo("Restart service snmp failed with error {}".format(e))
             raise click.Abort()
 
+#
+# 'bmp' group ('config bmp ...')
+#
+
+@config.group()
+@click.pass_context
+def bmp(ctx):
+    """BMP-related configuration"""
+
+#
+# 'enable' subgroup ('config bmp enable ...')
+#
+@bmp.group()
+@click.pass_context
+def enable(ctx):
```

Comment:
>enable [](http://example.com/codeflow?start=4&length=6)

Please add unit test and ensure each changed line covered.

Reply:
fixed. and moved into https://github.com/sonic-net/sonic-utilities/pull/3286 since this PR was not created on fork branch.